vendor/github.com/operator-framework/operator-lifecycle-manager/cmd/olm/cleanup.go

```diff
@@ -1,292 +0,0 @@
-package main
```

Comment:
We should factor extra-staging changes into their own commits to avoid potential conflicts later.